### ISSUE TITLE
Cannot run `bubbleprof --visualize-only`

### ISSUE THREAD

**Issue by jsumners** [nearform/node-clinic-bubbleprof#350]
```
❯ npx clinic bubbleprof --visualize-only clinic/13196.clinic-bubbleprof
Error: Cannot find module 'd3-color' from '/foo/project/node_modules/clinic/node_modules/@nearform/bubbleprof/visualizer/draw'
```

**Comment by goto-bus-stop** [comment]
Looks like we're missing an explicit dependency on d3-color. usually it had worked out fine because it is installed as a transitive dependency, but depending on how npm decides to lay out the tree, there is no guarantee that it'll actually be accessible from bubbleprof.

all we need is to add d3-color to our own package.json.

thanks for reporting this!

**Comment by goto-bus-stop** [comment]
Fixed in :package:  3.0.2. Thanks!
